lstchain/reco/r0_to_dl1.py

```diff
@@ -746,6 +762,11 @@ def r0_to_dl1(
         muon_output_filename = Path(dir, name)
         table = Table(muon_parameters)
         table.write(muon_output_filename, format='fits', overwrite=True)
+        
+        # close the interleaved output file and write metadata
+        if 'write_interleaved_events' in config.keys():
```

Comment:
Better you `if interleaved_writer is not None` since that doesn't need the config again.